.github/workflows/dockerhub.yml

```diff
@@ -89,12 +93,34 @@ jobs:
       - name: Determine Build Metadata
         id: build-metadata
         uses: usdot-fhwa-stol/actions/docker-build-metadata@main
+        
+      - name: Process Suffix in Output Docker Image tag (handle remove_suffix)
+        id: process-suffix-in-output-docker-tag
+        if: ${{ inputs.remove_suffix == true }}
+        run: |
+          # Extract the tag from the original output
+          ORIGINAL_TAG="${{ steps.docker-org-and-tag.outputs.docker_image_tag }}"
+          echo "Original tag: $ORIGINAL_TAG"
+          
+          # Remove everything after the first dash (if a dash exists)
+          if [[ "$ORIGINAL_TAG" == *-* ]]; then
+            BASE_TAG="${ORIGINAL_TAG%%-*}"
+            REMOVED_SUFFIX="${ORIGINAL_TAG#*-}"
```

Comment:
Could we add a quick code comment explaining exactly what %%-* does vs. %-* and why we picked each like first dash removal and after dash removal ?

Reply:
Added comments